### ISSUE TITLE
just a simple question

### ISSUE THREAD

**Issue by whislol** [NYAN-x-CAT/AsyncRAT-C-Sharp#230]
can you actually control the pc you have access to now? and if so how

**Comment by TomerGamerTV** [comment]
yes you can with remote control or something like that but its kida broken in its version so the only thing you can do is spectate him
